backend/main.go

```diff
@@ -352,7 +398,29 @@ func main() {
 			return
 		}
 	}
+
 	server := Server{store: store, storeTimeout: defaultStoreTimeout}
```

Comment:
It's a minor thing, but can we instantiate the `Server` *underneath* this block of code, and put our `auth` into a `var`, same as we do above for the `store`?

This is just a bit more consistent: we gather all the things we need to pass into the server and then we instantiate it - rather than instantiating the `Server` with half the things it needs and then setting the others later.

Reply:
Makes sense, done.